plugins/modules/cluster_proxy_addon.py

```diff
@@ -76,6 +76,18 @@
     IMP_ERR['k8s'] = {'error': traceback.format_exc(),
                       'exception': e}
 
+try:
+    import requests
```

Comment:
new dependencies? we need to update requirements.txt

Reply:
I added `requests` to requirements.txt